Specialization.cs

```diff
@@ -95,5 +108,22 @@ public static string GetCoreName(uint core)
             59, // Mirage
             66, // Virtuoso
         };
+
+        public static string GetEliteIconPath(uint elite, bool useTangoIcons)
+        {
+            var folder = useTangoIcons ? "professions_tango" : "professions_beige";
+            return @$"{folder}\{GetEliteName(elite)}.png";
+        }
+
+        public static string GetCoreIconPath(uint core, bool useTangoIcons)
+        {
+            var folder = useTangoIcons ? "professions_tango" : "professions_beige";
+            return @$"{folder}\{GetCoreName(core)}.png";
+        }
```

Comment:
I think these two functions would be better extracted elsewhere, as `GetIconPath(string iconName, bool useTangoIcons)`

Reply:
Agreed, but not sure where to extract them to. Maybe better after the MVP refactor?